_docs/ioos-metadata-profile-v1-2.md

```diff
@@ -246,34 +238,29 @@ This profile requires that all variables containing results from QARTOD tests be
+qartod_variable:references | CF | Published or web-based references that describe the data or methods used to produce it. Recommend URIs (such as a URL or DOI) for papers or other references.  For QARTOD, this should be used to refer by URI to a resource that describes the test configuration, parameters used, etc. | variable | **required**, if applicable
 
 #### Example
 
-Source: Big Carlos Pass ERDDAP Gold-Standard [dataset](http://erddap.sensors.axds.co/erddap/tabledap/big-carlos-pass-active.html).  **NOTE: this link is dead**
+Source: Morro Bay BS1 MET ERDDAP Gold-Standard [dataset](https://standards.sensors.ioos.us/erddap/info/morro-bay-bs1-met/index.html).
 
 ```
-air_temperature {
-  String ancillary_variables "air_temperature_can_name_this_whatever_you_want air_temperature_flat_line_test";
-  String long_name "Air Temperature";
-  String standard_name "air_temperature";
-  String units "degree_Celsius";
-}
-air_temperature_can_name_this_whatever_you_want {
-  String flag_meanings "PASS NOT_EVALUATED SUSPECT FAIL MISSING";
-  String flag_values "1, 2, 3, 4, 9";
-  String long_name "Air Temperature QARTOD Aggregate Flag";
-  String standard_name "aggregate_quality_flag";
-  String references "https://github.com/glos/glos-qartod/wiki/QARTOD-101";
-}
-air_temperature_flat_line_test {
-  String flag_meanings "PASS NOT_EVALUATED SUSPECT FAIL MISSING";
-  String flag_values "1, 2, 3, 4, 9";
-  String long_name "Air Temperature QARTOD Flat Line Test Flag";
-  String standard_name "flat_line_test_quality_flag";
-  String references "https://github.com/glos/glos-qartod/wiki/QARTOD-101";
-}
+  air_temperature {
+    String standard_name "air_temperature";
+    String long_name "Air Temperature";
+    String units "degree_Celsius";
+    String ancillary_variables "air_temperature_qc_agg air_temperature_qc_tests";
```

Comment:
@jessicaaustin I merged your commits outside of this PR, but the one thing I noticed that might help is to include a non-aggregate qc flag in this example.  In this case, it would be the `air_temperature_qc_tests` variable, but should that be renamed something else?   I think we're missing something that explains use of the aggregate flag alongside a standard flag.

Reply:
Latest version is available here now: https://mwengren.github.io/ioos-metadata/ioos-metadata-profile-v1-2.html

Also commit history: https://github.com/mwengren/ioos-metadata/commits/qartod_fixes

Do you want to add it in your same branch and I'll repeat the local merge again?  Or I can do it, either way.

Reply:
see https://github.com/mwengren/ioos-metadata/pull/3

We don't report individual tests as their own variable for the RA or sensor map ERDDAPs, so technically things like `air_temperature_qc_tests` fall outside of the QARTOD standards we're talking about. But I agree there needs to be a good example for people that do want to report each test individually.

Reply:
Ok, so you just share the rollup flag variable and don't report any individual test flags anywhere in the datasets?  

I was just re-reading that section of the profile, and as it reads now we're just requiring if they are present in the dataset they're labeled properly, we're not actually requiring them to accompany the rollup flag, so that's fine.  Just doublechecking the text says what we want it to:

"This profile requires that all variables containing results from QARTOD tests be referenced by ancillary variables, and those variables use a valid CF Standard Name to reflect the test performed (see qartod_variable:standard_name in the table below for a list of suitable standard names)"

Reply:
Yes, we just share the rollup flag variable in a standard way. We share the individual tests as well, but all smashed together in that one `_qc_tests` variable. (that variable is self-describing so I think it's technically compliant with CF, but it doesn't match any QARTOD standards we are working on with CF or the IOOS metadata profile)

And yeah, I agree the profile just says, if you are providing the rollup flag or individual qc tests, here is how to describe them. So I think it's ok.